build-logic-settings/build-environment/src/main/kotlin/gradlebuild.build-environment.settings.gradle.kts

```
@@ -20,11 +20,31 @@ import gradlebuild.basics.BuildEnvironmentService
 with(layout.rootDirectory) {
     gradle.lifecycle.beforeProject {
         val service = gradle.sharedServices.registerIfAbsent("buildEnvironmentService", BuildEnvironmentService::class) {
+            assert(project.path == ":") {
```

🤔  Is an assertion error what we use in such cases? Why not Kotlin's regular `check() {}` ?

```suggestion
            check(project.path == ":") {
```

---

Fixed